queries.sql

```diff
@@ -98,4 +98,48 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 
-SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
\ No newline at end of file
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+/* Query multiple tables */
+SELECT animals.name, owners.full_name 
+FROM animals 
+JOIN owners 
+ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Melody';
+
+SELECT animals.name AS animal_name, species.name AS species_type 
+FROM animals 
+JOIN species 
+ON animals.species_id = species.id 
+WHERE species.name = 'Pokemon';
+
+SELECT owners.full_name, animals.name 
+FROM owners 
+LEFT JOIN animals 
+ON animals.owner_id = owners.id;
+
+SELECT species.name, count(animals) 
+FROM animals 
+JOIN species 
+ON animals.species_id = species.id 
+GROUP BY species.name;
+
+SELECT animals.name 
+FROM animals 
+JOIN owners 
+ON animals.owner_id = owners.id 
+WHERE owners.full_name = 'Jennifer Orwell';
+
```

Comment:
- [ ] Here the query should be describing `List all Digimon owned by Jennifer Orwell.` but your query missed the `Digimons` part. kindly add `LEFT JOIN species ON animals.species_id = species.id` after line 30 and then `AND species.name = 'Digimon';` after line 131 to get the right query.